### DISCUSSION TITLE
Why can't I sign up for mastodon server called "Twingyeo.kr"?

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#25675] [Q&A]
I try mostly everything and still don't know how, i once had account on this server and it was way easier than now.

**Comment by StrangeMinds1960** [comment]
As far as I can tell, registrations are closed on that server

**Reply by ghost** [reply]
a few months ago I had access to register an account on this server, now a lot of people are switching to mastodon, especially among Koreans from twitter. Why is it closed now? and if i even see some going to this now even when you say it's closed.

**Reply by StrangeMinds1960** [reply]
The admin contact for this server is contact@unstabler.pl, send them an email
